XML/TestCases/FunctionalTests-SRIOV.xml

```diff
@@ -557,50 +519,49 @@
         <Priority>1</Priority>
     </test>
     <test>
-        <testName>SRIOV-BROADCAST</testName>
+        <testName>SRIOV-IPERF-STRESS</testName>
         <setupScript>.\Testscripts\Windows\SETUP-SR-IOV-Configure.ps1</setupScript>
-        <testScript>SRIOV-Broadcast.sh</testScript>
-        <files>.\Testscripts\Linux\SRIOV-Broadcast.sh,.\Testscripts\Linux\SR-IOV-Utils.sh,.\Testscripts\Linux\utils.sh</files>
+        <testScript>SRIOV-RUN-COMMON-TEST.ps1</testScript>
         <setupType>TwoVM2Host1Dep</setupType>
+        <AdditionalHWConfig>
+            <Networking>SRIOV</Networking>
+        </AdditionalHWConfig>
+        <files>.\Testscripts\Linux\SRIOV-Iperf-Stress.sh,.\Testscripts\Linux\SR-IOV-Utils.sh,.\Testscripts\Linux\utils.sh</files>
         <TestParameters>
+            <param>Remote_Script=SRIOV-Iperf-Stress.sh</param>
+            <param>Set_SSH=yes</param>
+            <param>Install_Dependencies=yes</param>
             <param>NIC_COUNT=1</param>
-            <param>VF_IP1=10.0.0.23</param>
-            <param>VF_IP2=10.0.0.43</param>
+            <param>VF_IP1=10.0.0.44</param>
+            <param>VF_IP2=10.0.0.64</param>
             <param>NETMASK=255.255.255.0</param>
             <param>Switch_Name=SRIOV_MLNX</param>
             <param>SECRET_PARAMS=(VM2Name,CheckpointName)</param>
         </TestParameters>
-        <Platform>HyperV</Platform>
+        <timeout>2400</timeout>
+        <Platform>Azure,HyperV</Platform>
         <Category>Functional</Category>
         <Area>SRIOV</Area>
         <Tags>sriov,network</Tags>
-        <Priority>1</Priority>
+        <Priority>2</Priority>
     </test>
     <test>
-        <testName>SRIOV-PCI-RESCIND</testName>
```

Comment:
@adriansuhov Have you renamed the test or removed it?

Reply:
@iamshital I remember this was renamed